src/linux/x86_64/kvm_cpu.rs

```diff
@@ -305,8 +306,14 @@ impl KvmCpu {
 		&mut self.vcpu
 	}
 
-	fn init(&mut self, entry_point: u64, stack_address: u64, cpu_id: u32) -> HypervisorResult<()> {
-		self.setup_long_mode(entry_point, stack_address, cpu_id)?;
+	fn init(
+		&mut self,
+		entry_point: u64,
+		stack_address: u64,
+		guest_address: u64,
```

Comment:
This could be GuestPhysicalAddresses, whilst we are on it, right?

Reply:
Partially resolved, `stack_address` and `entry_point` possibly unnecessary?